### ISSUE TITLE
[No Hope] starting scenarios are not overwritten by [Sky Island]

### ISSUE THREAD

**Issue by Mare-expectantem** [CleverRaven/Cataclysm-DDA#76740]
### Describe the bug

When inserting both [No Hope] and [Sky Island] into a World, Scenarios from both mods appear.

### Attach save file

N/A

### Steps to reproduce

- Start at the Title Screen
- Create a new World with both [No Hope] and [Sky Island]
- Attempt to Create a Custom Character
- See All of the Starting Scenarios

### Expected behavior

[Sky Island] should not allow other Scenarios to appear.

### Screenshots

![Screenshot](https://github.com/user-attachments/assets/a3228aad-3c88-45e8-9774-22fe7295dfe9)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.4894 (22H2)
- Game Version: 26de5a1 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Sky Island [skyisland],
    No Hope [no_hope]
]

**Comment by Standing-Storm** [comment]
What happens if you load Sky Island after No Hope?

**Comment by Mare-expectantem** [comment]
Only the Sky Island Scenario is available.

**Comment by Night-Pryanik** [comment]
I don't think we can or should guarantee total compatibility between each and any other mod. Especially for total conversion mods like Sky Island. I can only make a suggestion for mod authors that if they want to override other mods' settings, they should add a note in mod's description like "put this mod as low in mod list as possible".

**Comment by Standing-Storm** [comment]
> Only the Sky Island Scenario is available, which is what is supposed to happen. However Sky Island is almost always going to be further up the mod load order due to its positioning in the mod list. And even if it were pushed down, it likely wouldn't be a permanent fix.

There's no way to avoid questions of mod load order, unfortunately. I think as Night-Pryanik says, the best solution is just the classic "Place this mod at the bottom of your load order" note for mods whose changes absolutely need to take pre-eminence.

**Comment by thaelina** [comment]
This is no longer an issue, No Hope no longer has a scenario blacklist. However the actual issue can be solved by the mod by using a mod_interactions entry and reapply the relevant change again on top of that mod.
